frontend/src/component/admin/network/NetworkTrafficUsage/NetworkTrafficUsage.tsx

```diff
@@ -248,49 +269,75 @@ const NewNetworkTrafficUsage: FC = () => {
         }
     }, [data]);
 
+    // todo: extract this (and also endpoints info)
+    const [lastLabel, ...otherLabels] = Object.values(endpointsInfo)
+        .map((info) => info.label.toLowerCase())
+        .toReversed();
+    const requestTypes = `${otherLabels.toReversed().join(', ')}, and ${lastLabel}`;
+
     return (
         <ConditionallyRender
             condition={isOss()}
             show={<Alert severity='warning'>Not enabled.</Alert>}
             elseShow={
                 <>
                     <ConditionallyRender
-                        condition={includedTraffic > 0 && overageCost > 0}
+                        condition={
+                            (newPeriod.grouping === 'monthly' ||
+                                newPeriod.month ===
+                                    format(new Date(), 'yyyy-MM')) &&
+                            includedTraffic > 0 &&
+                            overageCost > 0
```

Comment:
Telling people they're "currently" using more data when the data we're measuring is from a past month seems misleading. So let's only show this if your chart includes data for the current month.